hydra-chain-observer/src/Hydra/ChainObserver.hs

```diff
@@ -51,37 +53,55 @@ import Ouroboros.Network.Protocol.ChainSync.Client (
 
 type ObserverHandler m = [ChainObservation] -> m ()
 
-data ChainObservation
-  = Tick
-      { point :: ChainPoint
-      , blockNo :: BlockNo
-      }
-  | HeadObservation
-      { point :: ChainPoint
-      , blockNo :: BlockNo
-      , onChainTx :: OnChainTx Tx
-      }
+data ChainObservation = ChainObservation
+  { point :: ChainPoint
+  , blockNo :: BlockNo
+  , onChainTx :: Maybe (OnChainTx Tx)
```

Comment:
@noonio Instead of https://github.com/cardano-scaling/hydra-explorer/pull/32 we should do just

```suggestion
  , observedTx :: Maybe (OnChainTx Tx)
```